src/lib/annotations/Annotator.js

```diff
@@ -287,11 +290,14 @@ class Annotator extends EventEmitter {
      * @return {void}
      */
     renderAnnotationsOnPage(pageNum) {
-        if (this.threads && this.threads[pageNum]) {
-            this.threads[pageNum].forEach((thread) => {
-                thread.show();
-            });
+        if (!this.threads) {
+            return;
         }
+
+        const pageThreads = this.getThreadsOnPage(pageNum);
```

Comment:
Could this be setup so that if it is given a certain value, it renders ALL annotations?

Reply:
I'm not sure what you mean? We are currently rendering all annotations on that page

Reply:
My bad, nvm